test/HiGlassComponentTest.js

```diff
@@ -2509,15 +2511,17 @@ describe('Simple HiGlassComponent', () => {
     });
 
     it('Selects mm9', (done) => {
-      const dropdownButton = hgc.find('.assembly-pick-button');
+      hgc.find('.assembly-pick-button');
```

Comment:
I don't think this does anything on it's own. If it's not used later, it can be deleted.

Reply:
👍